examples/rmg/minimal_surface/input.py

```diff
@@ -1,67 +1,123 @@
 # Data sources
 database(
-    thermoLibraries = ['primaryThermoLibrary'],
-    reactionLibraries = [],
+    thermoLibraries=['surfaceThermoPt111', 'primaryThermoLibrary', 'thermo_DFT_CCSDTF12_BAC','DFT_QCI_thermo'], # 'surfaceThermoPt' is the default. Thermo data is derived using bindingEnergies for other metals 
+    reactionLibraries = [('Surface/CPOX_Pt/Deutschmann2006_adjusted', False)], # when Ni is used change the library to Surface/Deutschmann_Ni 
     seedMechanisms = [],
     kineticsDepositories = ['training'],
-    kineticsFamilies = 'default',
+    kineticsFamilies = ['surface', 'default'],
     kineticsEstimator = 'rate rules',
 )
-
-# List of species
+catalystProperties(
+    bindingEnergies = {
+        'H': (-2.75368,'eV/molecule'),
+        'C': (-7.02516,'eV/molecule'),
+        'N': (-4.63225,'eV/molecule'),
+        'O': (-3.81153,'eV/molecule'),
+    },
+    surfaceSiteDensity=(2.483e-9, 'mol/cm^2'),
+    coverageDependence=False
+)
 species(
-    label='ethane',
+    label='CH4',
     reactive=True,
-    structure=SMILES("CC"),
+    structure=SMILES("C"),
 )
-
-# Reaction systems
-simpleReactor(
-    temperature=(1350,'K'),
-    pressure=(1.0,'bar'),
-    initialMoleFractions={
-        "ethane": 1.0,
-    },
-    terminationConversion={
-        'ethane': 0.9,
-    },
-    terminationTime=(1e6,'s'),
+species(
+   label='O2',
+   reactive=True,
+   structure=adjacencyList(
+       """
+1 O u1 p2 c0 {2,S}
+2 O u1 p2 c0 {1,S}
+"""),
 )
-
-simpleReactor(
-    temperature=(1000,'K'),
-    pressure=(1.0,'bar'),
-    initialMoleFractions={
-        "ethane": 1.0,
+species(
+    label='N2',
+    reactive=False,
+    structure=SMILES("N#N"),
+)
+species(
+    label='X',
+    reactive=True,
+    structure=adjacencyList("1 X u0"),
+)
+# added for training
+species(
+    label='CO2X',
+    reactive=True,
+    structure=adjacencyList(
+    """
+    1 O u0 p2 c0 {3,D}
+    2 O u0 p2 c0 {3,D}
+    3 C u0 p0 c0 {1,D} {2,D}
+    4 X u0 p0 c0
+    """),
+)
+species(
+    label='COX',
+    reactive=True,
+    structure=adjacencyList(
+    """
+    1 O u0 p2 c0 {2,D}
+    2 C u0 p0 c0 {1,D} {3,D}
+    3 X u0 p0 c0 {2,D}
+    """),
+)
+species(
+    label='OX',
+    reactive=True,
+    structure=adjacencyList(
+    """
+    1 O u0 p2 c0 {2,D}
+    2 X u0 p0 c0 {1,D}
+    """),
+)
+# If you would like to forbid the bidentate form of absorbed CO2 from your model,
+# use the following `CO2_bidentate` forbidden structure
+# forbidden(
+#     label='CO2_bidentate',
+#     structure=SMILES("O=C(*)O*"),
+# )
+#----------
+# Reaction systems
+surfaceReactor(
+    temperature=(1000, 'K'),
+    initialPressure=(1.0, 'bar'),
+    initialGasMoleFractions={
+        "CH4": 0.15,
+        "O2": 0.15,
+        "N2": 0.7,
     },
-    terminationConversion={
-        'ethane': 0.9,
+    initialSurfaceCoverages={
+        "X": 1.0,
     },
-    terminationTime=(1e6,'s'),
+    surfaceVolumeRatio=(1.e5, 'm^-1'),
+    terminationConversion = { "CH4": 0.95,},
```

Comment:
For style consistency

```suggestion
    terminationConversion={"CH4": 0.95,},
```